docker-compose.yml

```diff
@@ -18,10 +18,22 @@ services:
     container_name: performance_manager
```

Comment:
Should probably update this image to match our AWS RDS version:

`image: postgres:13.4`